normalize.js

```diff
@@ -111,11 +126,8 @@ define(function() {
     
     return out.substr(0, out.length - 1);
   };
-  
-  var normalizeCSS = function(source, fromBase, toBase) {
 
-    fromBase = removeDoubleSlashes(fromBase);
-    toBase = removeDoubleSlashes(toBase);
+  var normalizeCSS = function(source, fromBase, toBase) {
```

Comment:
@jcppman OMG, this is so my bad :) Thanks. I'm going to merge this PR.

But before that, could u please let me know what test u was relying on during this fix?